app/views/play/menu/InventoryModal.coffee

```diff
@@ -463,7 +463,7 @@ module.exports = class InventoryModal extends ModalView
         delete equipment[slot]
 
   calculateRequiredGearPerSlot: ->
-    return {} if me.isStudent() and not application.getHocCampaign()
+    return {} if me.isStudent() and not application.getHocCampaign() and not me.showHeroAndInventoryModals()
```

Comment:
Should rename this.  Doesn't make sense to check that we didn't show something in the code for thing we didn't want to show.